### PR TITLE
Official NGINX image supported by NGINX, Inc.

### PR DESCRIPTION
**Pull request by cpretzer**
We have created an official nginx image which will continue to maintain and support for the mesosphere universe.

### CODE REVIEW

**Comment by joerg84** [comment]
@cpretzer Just curious: is there a reason not to update the other nginx package with this image?

**Comment by cpretzer** [comment]
@joerg84 the main reason was to preserve the version that the folks at bitnami created. We are happy to take that over, if they no longer want to support it.

File: repo/packages/N/nginx-official/0/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "version": "1.13.1",
+  "scm": "http://hg.nginx.org/nginx/",
+  "description": "Official nginx package",
+  "maintainer": "support@nginx.com",
+  "tags": ["proxy", "web-server", "cache"],
+  "preInstallNotes": "This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Preview packages should never be used in production! Preparing to install nginx.",
+  "postInstallNotes": "Nginx is installed.",
+  "postUninstallNotes": "Nginx is uninstalled successfully.",
+  "website": "https://www.nginx.org",
+  "minDcosReleaseVersion": "1.0.8",
```
Comment (ryadav88) [inline repo/packages/N/nginx-official/0/package.json]: should be 1.8.0

File: repo/packages/N/nginx-official/0/resource.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://s3-us-west-1.amazonaws.com/nginx-universe/N-hex-48x48.png",
+    "icon-medium": "https://s3-us-west-1.amazonaws.com/nginx-universe/N-hex-96x96.png",
+    "icon-large": "https://s3-us-west-1.amazonaws.com/nginx-universe/N-hex-256x256.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "nginx-docker": "nginx:latest"
```
Comment (ryadav88) [inline repo/packages/N/nginx-official/0/resource.json]: use tagged version of a image.

File: repo/packages/N/nginx-official/0/package.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "packagingVersion": "4.0",
+  "name": "nginx-official",
```
Comment (ryadav88) [inline repo/packages/N/nginx-official/0/package.json]: change `name` to `nginx` and also update the subdirectory version to 7

**Review by ryadav88 — CHANGES_REQUESTED** [review]
@cpretzer please review and update the example in github.com/dcos/examples

**Comment by OhRobin** [comment]
installation works with the example listed here https://github.com/dcos/examples/tree/master/nginx/1.9

**Comment by ryadav88** [comment]
@cpretzer any update here?

**Comment by cpretzer** [comment]
@ryadav88, thanks for checking in. I'm planning to have this updated in the next few weeks.

**Comment by ryadav88** [comment]
@cpretzer any update here?

**Comment by ryadav88** [comment]
closing because of inactivity. Please reopen if youre still working on it.

**Pull request closed without merging**